pandas/plotting/_core.py

```diff
@@ -1749,11 +1749,8 @@ def _find_backend(backend: str):
     import pkg_resources  # Delay import for performance.
 
     for entry_point in pkg_resources.iter_entry_points("pandas_plotting_backends"):
-        if entry_point.name == "matplotlib":
-            # matplotlib is an optional dependency. When
-            # missing, this would raise.
-            continue
-        _backends[entry_point.name] = entry_point.load()
+        if entry_point.name == backend:
+            _backends[entry_point.name] = entry_point.load()
```

Comment:
One thing I'm not entirely sure of, at the moment `_find_backend` should never be called with `backend='matplotlib'`, as this case is handled by `_get_plot_backend` alone. However maybe the `if entry_point.name == "matplotlib":` should be kept in case this function is reused some other way in the future.

Reply:
Resolved with e84b305